model/src/main/kotlin/config/ErrorResolutionReason.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model.config
+
+enum class ErrorResolutionReason {
+    /**
+     * The error is due to the way a package is built.
+     */
+    BUILD_TOOL_OF,
```

Comment:
We have the same enum member in `ProjectExcludeReason` and `ScopeExcludeReason` already, which maybe creates confusion when to use which. What's an example for this?

Reply:
I think it's good to reuse the same names, makes it easier to remember.
One example would be a Gradle project using a `flatDir` repository. This will generate an analyzer error because we do not support this, which could be resolved using `BUILD_TOOL_OF`.

Reply:
Then let's use `BUILD_TOOL_ISSUE` [as suggested by Thomas](https://github.com/heremaps/oss-review-toolkit/issues/731#issuecomment-412806641) but extend the docs to e.g. "The error originates from the build tool used by the project".

Reply:
Done.